shared/templates/service_enabled_guard_var/oval.template

```diff
@@ -0,0 +1,170 @@
+<def-group>
+
+{{%- set package_installed_test_id = "test_service_" + SERVICENAME + "_package_" + PACKAGENAME + "_installed" -%}}
+{{%- set variable_value_test_id = _RULE_ID + "_test_variable_" + VARIABLE -%}}
+{{% if OPERATION is defined %}}
+  {{%- set variable_value_op = OPERATION -%}}
+{{% else %}}
+  {{%- set variable_value_op = "equals" -%}}
+{{% endif %}}
+{{% if target_oval_version >= [5, 11] %}}
```

Comment:
Currently, the target version is always 5.11. We dropped 5.10 support a long time ago.

Reply:
dropped outdated code thanks 🙇